File: hyesu/문자열/1764.py
import sys
input = sys.stdin.readline

n, m = map(int, input().split())

listens = []
sees = []

for _ in range(n):
    listens.append(input())

for _ in range(m):
    sees.append(input())

ans = list(set(listens) & set(sees))
ans.sort()
print(len(ans))
print(''.join(ans), end='')

File: hyesu/SwiftAlgorithm/SwiftAlgorithm/S4/1764.swift
//
//  1764.swift
//  SwiftAlgorithm
//
//  Created by 김혜수 on 2023/02/15.
//

import Foundation

var nums = readLine()!.split(separator: " ").map { Int($0)! }
let n: Int = nums[0]
let m: Int = nums[1]
var listens: [String] = []
var watchs: [String] = []

for _ in 0..<n {
    listens.append(readLine()!)
}

for _ in 0..<m {
    watchs.append(readLine()!)
}

var ans = Array(Set(listens).intersection(Set(watchs)))
print(ans.count)
ans.sort()
print(ans.joined(separator: "\n"))
